evil-vars.el

```diff
@@ -518,7 +518,7 @@ ubiquity of prefix arguments."
                (define-key evil-insert-state-map (kbd "C-u") 'evil-delete-back-to-indentation))))))
 
 (defcustom evil-want-C-w-delete t
-  "Whether `C-w' deletes a word in Insert state."
+  "Whether `C-w' deletes a word in Insert/Ex/search state."
```

Comment:
Elsewhere, the patch has

``` elisp
(define-key evil-ex-completion-map "\C-w" nil)
```

So for now the documentation shouldn't mention support for Ex state.

Reply:
fixed.

Reply:
But it means if `evil-want-C-w-delete` is not nil, C-w in `evil-ex-completion-map` bind to `backward-kill-word`
if not C-w would call `kill-region` or whatever command  binding  in `global-map`.

```
(if evil-want-C-w-delete
    (define-key evil-ex-completion-map "\C-w" 'backward-kill-word)
    (define-key evil-ex-completion-map "\C-w" nil))
```

Reply:
Oh sorry, you're correct. I had the yanking functionality in mind when I commented on this, but you had it right the first time.